Source/DafnyCore/Options/ProjectFile.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Runtime.Serialization;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.FileSystemGlobbing;
+using Microsoft.Extensions.FileSystemGlobbing.Abstractions;
+using Tomlyn;
+
+namespace Microsoft.Dafny; 
+
+public class ProjectFile {
+  [IgnoreDataMember]
+  public Uri Uri { get; set; }
```

Comment:
I'm not sure what change your comment is asking for. If it's a local path or a URL, isn't a `Uri` the right type? My impression is that `Uri` is great for paths since it's more platform independent than `string`.

Reply:
Ah if this is just a clean implementation for a local path that's fine too. But we should probably double check the protocol is always `file:///`.

Reply:
Added a check to make sure passed project files exist on disk.